tests/functional/object/rgw/test_obc_quota.py

```diff
@@ -117,3 +122,75 @@ def test_obc_quota(
                 logger.error("Copy objects to bucket failed unexpectedly!!")
         else:
             logger.info(f"New quota {new_quota_str} got applied!!")
+
+    @bugzilla("2188032")
+    @polarion_id("OCS-6178")
+    @pytest.mark.parametrize(
+        argnames="amount,interface,quota",
+        argvalues=[
+            pytest.param(
+                *[1, "RGW-OC", {"maxObjects": "10"}],
+                marks=[
+                    tier2,
+                ],
+            ),
+        ],
+    )
+    def test_obc_quota_full_alert(
+        self,
+        rgw_bucket_factory,
+        rgw_obj_session,
+        awscli_pod_session,
+        test_directory_setup,
+        threading_lock,
+        amount,
+        interface,
+        quota,
+    ):
+        """
+        This test will verify the prometheus alerts
+        when the OBC quota is reached
+
+        """
+
+        # create the bucket
+        bucket_name = rgw_bucket_factory(amount, interface, quota=quota)[0].name
+        logger.info(f"created rgw bucket {bucket_name} with quota {quota}")
+
+        # fill the bucket with about 90% of maxObjects
+        number_of_objects = int(quota["maxObjects"])
+        write_random_test_objects_to_bucket(
+            awscli_pod_session,
+            bucket_name,
+            test_directory_setup.origin_dir,
+            amount=(number_of_objects * 90) // 100,
```

Comment:
this can result in less than 90%.
For example:
if `number_of_objects` is 31, `amount` will be:
(31*90) // 100 = 27
27 are 87% of 31

Reply:
@ebenahar here we are not considering the decimal value. 90% of 31 will 27.9 not 27. but in this case we are ignoring the decimal value. We dont need 90%, we only need (80- 85%)